src/components/Footer/Footer.module.css

```diff
@@ -1,30 +1,50 @@
 .footer {
-    display: flex;
-    flex-flow: column wrap;
-    align-items: center;
-    height: 120px;
+  display: flex;
+  flex-flow: row;
+  align-items: center;
 }
 
-.button__wrap{
-    display: flex;
-    flex-flow: row wrap;
-    justify-content: center;   
-    margin-top: 10px; 
-    height: 80px;   
+.button__wrap {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
 }
 
 .button {
-    padding: 5px;
-    background: white;
-    border: #0000FF 1px solid;
-    border-radius: 10px;
-    margin: 5px;
-    cursor: pointer;
-    box-shadow: none;
+  background: white;
+  margin: 5px;
+  padding: 5px;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  font: normal normal 16px/18px Roboto, sans-serif;
+  display: flex;
+  flex-flow: row nowrap;
 }
 
-.button:hover {    
-    transition: .7s;
-    border: #10FF58 2px solid;
-    border-radius: 10px;
-}
\ No newline at end of file
+.button:hover {
+  background: #356eff;
+  border-radius: 23px;
+  color: #fff;
+}
+
+.count_item {
+  padding-left: 10px;
+  color: #356eff;
+  font-weight: bold;
+  margin: 0;
+}
+
+.button_active .count_item {
+  color: #fff;
+}
+
+.button:hover .count_item {
+  color: #fff;
+}
+
+.button_active {
+  background: #356eff;
+  border-radius: 23px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)